cmd/main.go

```diff
@@ -1,7 +1,7 @@
 package main
```

Comment:
Shouldn't this be under the cmd/log-collector dir?

Reply:
Yes.. we can keep it that way no issues with that. If followed that, dir structure should be like - 
- cmd
  - log-collector 
    - cmd/log-collector.go
    - main.go 

There's supposed to be a single log-collector executable in `cmd` so did not bother to create separate and nested dir structure like above.

One thing that should be followed is executable(main.go) and commands(log-collector.go) file should be kept in separate dir, in order to keep them in different go packages.  
[ref](https://github.com/spf13/cobra#getting-started).

Reply:
@pallav-trilio I am ok with the above structure suggested by @msurudwad

Reply:
@pallav-trilio also need to update `.goreleaser.yml` for updated `main.go` path. Make sure to run `make build-log-collector` before git push.

Reply:
Alright

Reply:
resolved